src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -105,36 +159,51 @@ public ConnectionBuilder<TSourceType> Bidirectional()
             return this;
         }
 
+        /// <inheritdoc cref="FieldBuilder{TSourceType, TReturnType}.Name(string)"/>
```

Comment:
`inheritdoc` here? `ConnectionBuilder<TSourceType>` does not inherit from `FieldBuilder`.

Reply:
Sure why not?  It's just copying the one-line description, and the description is still accurate -- it's the name of the field.

Reply:
ok